### DISCUSSION TITLE
.bnb .arb ENS

### DISCUSSION THREAD

**Discussion by Markcode1992** [rainbow-me/rainbowkit#1346] [Ideas]
Hello, are you guys working on integrating .bnb and .arb domains to show up in the address field?

**Comment by DanielSinclair** [comment]
Not currently. We primarily rely on Wagmi's `useEnsName()` hook to resolve names. This may be something that could be override via the RainbowKit API so that dApps can implement their own resolvers. Would love to hear if you have ideas/thoughts on what that could look like. We're also open to a PR!

**Comment by Galya01** [comment]
I cant work there

**Reply by Markcode1992** [reply]
What do you mean
